rclpy/src/rclpy/node.hpp

```diff
@@ -0,0 +1,49 @@
+// Copyright 2021 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef RCLPY__NODE_HPP_
+#define RCLPY__NODE_HPP_
+
+#include <pybind11/pybind11.h>
+
+#include <string>
```

Comment:
Unused header?

Reply:
I forgot to clean up headers after the copy/pasting. Sorry about that!

Reply:
Still unused?